library/CMakeLists.txt

```diff
@@ -684,36 +684,41 @@ if(TINYSPLINE_CXX_AVAILABLE AND TARGET_SUPPORTS_SHARED_LIBS)
 
 		# PHP
 		if(TINYSPLINE_WITH_PHP)
-			find_package(PHP)
+			find_package(PHP COMPONENTS Development)
 			if(PHP_FOUND)
-				set(TINYSPLINE_PHP_VERSION "php5")
-				if(${PHP_VERSION_NUMBER} MATCHES "^7")
-					set(TINYSPLINE_PHP_VERSION "php7")
-				endif()
-				if(${TINYSPLINE_PHP_VERSION} STREQUAL "php7" AND ${SWIG_VERSION} VERSION_LESS 3.0.11)
-					message(STATUS "PHP 7 requires Swig 3.0.11 or later")
-					# Clear PHP related variables used in summary.
-					set(PHP_FOUND FALSE)
-					unset(PHP_VERSION_STRING)
-					unset(PHP_INCLUDE_DIRS)
 				else()
```

Comment:
I think something went wrong here. It looks like PHP is only added if PHP was not found :).

Reply:
...Oops.  Heheh, my bad!